<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualChannelDialog</class>
 <widget class="QDialog" name="VirtualChannelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Virtual Channels</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxClipboard">
     <property name="title">
      <string>Clipboard</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="enableClipboardCheckBox">
        <property name="text">
         <string>Enable clipboard sharing</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="clipboardTextCheckBox">
        <property name="text">
         <string>Share text</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="clipboardImageCheckBox">
        <property name="text">
         <string>Share images</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="clipboardFileCheckBox">
        <property name="text">
         <string>Share files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAudio">
     <property name="title">
      <string>Audio</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="enableAudioCheckBox">
        <property name="text">
         <string>Enable audio redirection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="audioPlaybackCheckBox">
        <property name="text">
         <string>Audio playback</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="audioRecordingCheckBox">
        <property name="text">
         <string>Audio recording</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxDevices">
     <property name="title">
      <string>Device Redirection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="enableDriveRedirectionCheckBox">
        <property name="text">
         <string>Enable drive redirection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enablePrinterRedirectionCheckBox">
        <property name="text">
         <string>Enable printer redirection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enablePortRedirectionCheckBox">
        <property name="text">
         <string>Enable port redirection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 